### ISSUE TITLE
Listing some good template/example/top-notch presentations

### ISSUE THREAD

**Issue by danyill** [asciidoctor/asciidoctor-reveal.js#262]
A few examples of top-notch open source asciidoctor-reveal.js presentations would help the casual user get started and provide a good base for hacking a presentation together. If people are like me they don't have time to do the infrastructure work because they started their presentation well behind schedule and just want something to start from which works and hopefully shows a good syntax range.

A couple of suggestions:

* https://github.com/melix/gradle-presentation-template
* https://gitlab.com/jorge-aguilera/presentation-template

I'd like to add a Gulp+Live Reload template as well, perhaps in some time (mine is not suitable for publication at present).

Could we gather a few and include them in the readme?

**Comment by bentolor** [comment]
Not sure if top notch, but I'm really suprised that I sticked with asciidoctor-revealjs until the _very end_. At first my idea was to use it only for drafting and then to improve manually.

Nevertheless my presentation has yet _quite_ a few hacky/nasty parts and it also requires #267 #268 #269 to be merged before it will start to works.

All kind of reveal-js crazyness like backgroudn movies, fragments etc. included. Best watched in Firefox: Chrome sometimes fails with the vids. 

[So if you ever were interested into learning the news from Java 9 to 13 and up this presentation is for you ;-)](https://bentolor.github.io/java9to13/)

[asciidoctor-revealjs Source](https://github.com/bentolor/java9to13/tree/master/docs)

**Comment by obilodeau** [comment]
Yes @danyill, it's a good idea. Feel free to start a README section in a PR or wait for me to get to it eventually.

> [So if you ever were interested into learning the news from Java 9 to 13 and up this presentation is for you ;-)](https://bentolor.github.io/java9to13/)

Oh my god, that's a very pretty presentation! Congrats! You inspire me to step up my game now :smile:. Videos in backgrounds for transitions or topic switches is such a good idea!

**Comment by bentolor** [comment]
I think ideally visual guide/catalog (including screenshots of the result) would be helpful. What would be your stance about storage location for screenshots @obilodeau?

Or even better: how about introducing/showing of asciidoctor-reveal.js with a homepage of its output: _asciidoctor-reveal.js presenting „asciidoctor-reveal.js“_

**Comment by obilodeau** [comment]
I like what was done in PR #268 

> Or even better: how about introducing/showing of asciidoctor-reveal.js with a homepage of its output: _asciidoctor-reveal.js presenting „asciidoctor-reveal.js“_

Yes! That too would be good. I don't see myself tackling this relatively soon so if anyone wants to step up and make happen it would be appreciated. I'll happily review, comment and offer suggestions.

**Comment by ggrossetie** [comment]
Revisiting this as part of a pre-release triage pass. There's good material in the thread already (the Java 9-13 presentation, the PR #268 idea of an asciidoctor-reveal.js-presenting-itself homepage), but nothing has landed and the issue has stalled without a concrete next step.

As @obilodeau said earlier: feel free to open a PR adding a README/docs section listing a few example presentations (screenshots included, if we can agree on where to host them) — that's a much more actionable path than tracking suggestions here indefinitely.

Closing this one out; happy to review a PR whenever someone wants to put one together.
